playbooks/roles/common-server-init/tasks/misc.yml

```diff
@@ -17,17 +17,25 @@
     state: link
     force: true
 
-- name: add Opencraft repository
-  apt_repository:
-    repo: "{{ COMMON_SERVER_OPENCRAFT_REPO }}"
-    state: present
-
 - name: add Opencraft repository key
   apt_key:
     keyserver: "{{ COMMON_SERVER_OPENCRAFT_REPO_KEY_SERVER }}"
     id: "{{ COMMON_SERVER_OPENCRAFT_REPO_KEY_ID }}"
     state: present
 
+- name: add Opencraft repository
+  apt_repository:
+    repo: "{{ COMMON_SERVER_OPENCRAFT_REPO }}"
+    state: present
+
+- name: set apt preferences to prioritize the opencraft repo
+  copy:
+    content: |
+            Package: chkrootkit
+            Pin: Origin "debs.opencraft.com"
```

Comment:
@hchandad, can you please factor out debs.opencraft.com into its own variable (like COMMON_SERVER_OPENCRAFT_REPO_DOMAIN) and use it here?  This way we avoid problems if by any chance the domain changes, some day.

Reply:
Done !